client/ionic/src/pages/ProtectYourself.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {
+  IonContent,
+  IonHeader,
+  IonItem,
+  IonImg,
+  IonPage,
+  IonToolbar,
+  IonSlides,
+  IonSlide,
+  IonCard,
+  IonCardContent,
+} from '@ionic/react';
+import 'tachyons';
+
+const ProtectYourself: React.FC = () => {
+  return (
+    <IonPage className="pa3">
+      <IonHeader>
+        <IonToolbar>
+          <IonItem href="/menu">
+            <IonImg
+              className="w-80 center h3"
+              src="assets/identity/who-logo-rgb.png"
+            />
+          </IonItem>
+        </IonToolbar>
+      </IonHeader>
+      <IonContent>
+        <IonSlides pager={true}>
+          <IonSlide>
+            <IonCard>
+              <IonCardContent className="pb3 tc ph5">
+                <IonImg
+                  className="w-80 center"
+                  src="assets/content/img/protect_yourself/wash_hands.png"
```

Comment:
do we want kebab case for dir urls too?

Reply:
that's my preference, too, but again open to other's preferences

Reply:
done